lib/_http_server.js

```diff
@@ -557,6 +559,17 @@ Server.prototype.close = function() {
   ReflectApply(net.Server.prototype.close, this, arguments);
 };
 
+Server.prototype[SymbolAsyncDispose] = async function() {
+  const { promise, reject, resolve } = createDeferredPromise();
```

Comment:
Instead it could be `util.promisify(this.close).call(this)` probably? (with primordials?)